### ISSUE TITLE
Reticulated platelets increased on peripheral blood smear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10683]
**Preferred term label:**
Reticulated platelets increased on peripheral blood smear

**Synonyms**
immature platelets

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0012524

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/620776 "Peripheral blood showed increased reticulated platelets"

NOTE: We also have a pending term request for "Reticulated platelets decreased on peripheral blood smear"

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
This parameter seems to be largely the same as Immature platelet fraction
https://pmc.ncbi.nlm.nih.gov/articles/PMC8150321/
In any case, it seems it is "always" measure in peripheral blood but not necessarily on a smear (e.g. flow cytometry or other automated devices may be used).
So my question is if there is a reason to be this specific: do we need to specific reticulated platelets (as opposed to "young" or "immature" platelets) and do we need to specific blood smear?

**Comment by MickeySegal** [comment]
We included immature platelets as a synonym and would be glad to use that for both the "increased" term and the "decreased" term.

We don't need to specify how this is ascertained.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001238
- label: Elevated circulating reticulated platelet count
- parents: [Abnormal platelet count (HP:0011873)](https://hpo.jax.org/app/browse/term/HP:0011873)
- pmid: PMID:24807169
- def: The number of reticulated platelets per volume of blood in the peripheral circulation is above the upper limit of normal.
- comment: Reticulated platelets (RPs) are young, hyper-reactive platelets that are newly released from the bone marrow. RPs are thus immature platelets circulating in blood; they reflect the activity of megakaryopoiesis in the bone marrow.They are larger and contain more RNA compared to older platelets. In comparison to more mature platelets, they exhibit a significantly higher thrombogenicity and are known to be elevated in patients with an increased platelet turnover such as, diabetics and after acute myocardial infarction.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10683
- synonyms: 


and

Adding new term: 
- ID: HP:6001239
- label: Reduced circulating reticulated platelet count
- parents: [Abnormal platelet count (HP:0011873)](https://hpo.jax.org/app/browse/term/HP:0011873)
- pmid: PMID:24807169
- def: The number of reticulated platelets per volume of blood in the peripheral circulation is below the lower limit of normal.
- comment: Reticulated platelets (RPs) are young, hyper-reactive platelets that are newly released from the bone marrow. RPs are thus immature platelets circulating in blood; they reflect the activity of megakaryopoiesis in the bone marrow.They are larger and contain more RNA compared to older platelets. In comparison to more mature platelets, they exhibit a significantly higher thrombogenicity and are known to be elevated in patients with an increased platelet turnover such as, diabetics and after acute myocardial infarction.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10683
- synonyms: Decreased immature platelet fraction(NONE)
